udata_hydra/routes/__init__.py

```diff
@@ -0,0 +1,27 @@
+from aiohttp import web
+
+from udata_hydra.routes.checks import get_all_checks, get_latest_check
+from udata_hydra.routes.resources import create_resource, delete_resource, update_resource
+from udata_hydra.routes.status import get_crawler_status, get_stats, get_worker_status
+
+# routes = web.RouteTableDef()
+routes: list = []
+
+routes.extend(
+    [
+        # Routes for checks
+        web.get("/api/checks/latest/", get_latest_check),
+        web.get("/api/checks/all/", get_all_checks),
+        # Routes for resources
+        web.post("/api/resources/", create_resource),
+        web.put("/api/resources/", update_resource),
+        web.delete("/api/resources/", delete_resource),
+        web.post("/api/resource/created/", create_resource),  # TODO: legacy, to remove
```

Comment:
I think you should probably update the Readme to document the new routes and the old ones deprecation!

Reply:
Done